webapp/components/notifications/NotificationMenu/NotificationMenu.vue

```diff
@@ -29,13 +41,37 @@ export default {
   },
   data() {
     return {
+      displayedNotifications: [],
       notifications: [],
     }
   },
   props: {
     placement: { type: String },
   },
+  watch: {
+    notifications: {
+      immediate: true,
+      handler(lastQueriedNotifications) {
+        if (lastQueriedNotifications && lastQueriedNotifications.length > 0) {
+          lastQueriedNotifications.forEach(updatedListNotification => {
```

Comment:
Use `filter` instead of `forEach`:

```js
const newNotifications = lastQueriedNotifications.filter(new => {
    return !this.displayedNotifications.find(old => sameSame(new, old))
])
```

Reply:
I like this! 😍
So we make a sports out of finding the shortest coding.
I just like to know the headline …